internal/api/verify.go

```diff
@@ -89,7 +89,14 @@ func (a *API) verifyGet(w http.ResponseWriter, r *http.Request) error {
 		grantParams models.GrantParams
 		err         error
 		token       *AccessTokenResponse
+		authCode    string
 	)
+	var flowType models.FlowType
+	if firstN(params.Token, 5) == PKCEPrefix {
+		flowType = models.PKCEFlow
+	} else {
+		flowType = models.ImplicitFlow
+	}
```

Comment:
TBH I'd rather this uses `strings.HasPrefix` instead.

```suggestion
	if strings.HasPrefix(params.Token, PKCEPrefix) {
		flowType = models.PKCEFlow
	} else {
		flowType = models.ImplicitFlow
	}
```